localstack/services/opensearch/cluster.py

```diff
@@ -118,6 +120,67 @@ def build_cluster_run_command(cluster_bin: str, settings: CommandSettings) -> Li
     return [cluster_bin] + cmd_settings
 
 
+def register_cluster(host, path, forward_url, custom_endpoint) -> List[Rule]:
+    from localstack.services.edge import ROUTER
```

Comment:
The local import should not be necessary here.

Reply:
:+1: